Telegram Node: sendChatAction supports topics in supergroups

neoromantic opened the pull request:
## Summary

Currently, in sendChatAction there's no method to send chat action into a topic in group.

This PR adds "additional fields" option with "message thread id" field, that allows specifying topic to send chat action to.

<img width="345" alt="image" src="https://github.com/n8n-io/n8n/assets/44672/26825d68-2ec6-4763-96b3-c94fa7df9585">

## Review / Merge checklist
- [x] PR title and summary are descriptive. 
- [x] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.

neoromantic committed:
sendChatAction supports topics in supergroups in Telegram Node

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/n8n-io/n8n?pullRequest=9382) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/n8n-io/n8n?pullRequest=9382) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/n8n-io/n8n?pullRequest=9382) it.</sub>

neoromantic commented:
Is there a way I can help with considering this PR to be merged?

Joffcom commented:
Hey @neoromantic,

I have taken a look at your change, what that function does and what options the node has available and this change doesn't actually add anything to the `sendChatAction` API call.

Send Chat Action has 2 inputs the Chat ID and the Action, Both of these values look to be set and sent as part of the request. The `addAdditionalFields` function adds specifc fields to the request if there are any to add.

What is it that this PR is meant to achieve and did you test it before raising it?

neoromantic committed:
Merge remote-tracking branch 'upstream/master' into patch-1

neoromantic committed:
sendMessageAction operation supports topics in supergroups

neoromantic commented:
Hey, thank you for taking your time. I have to come clean: I did not do it properly, did not test, just made an assumption that this is one-liner, and wasted some of your time. For that, I am sorry.

I've spent some time and done it properly now. 

The purpose of change: support sending chat action in topics in Telegram. It comes down to adding one parameter to a query. It was almost done in existing code, but had not been enabled properly.

Currently there is no option to specify Message Thread ID (which is usually shown in Additional fields in other operations), there are not "additional fields" for sendChatAction.

My change: 
  - added "Additional fields" with "Message Thread ID" to sendChatAction operation
  - fixed logic in addAdditionalFields to skip over replyMarkup logic if operation is sendChatAction

I've tested it in production settings in real telegram groups and it works.

Relevant Telegram docs:  https://core.telegram.org/bots/api#sendchataction

Joffcom commented:
@neoromantic perfect, I will take a look at this on my next review session.

neoromantic committed:
Merge branch 'master' into patch-1

neoromantic commented:
@Joffcom I've added another PR too: https://github.com/n8n-io/n8n/pull/9531/files

Appreciate if you check that out as well, when you have time.
